test.rb

```diff
@@ -0,0 +1 @@
+path = Pathname.new("foo/bar/baz")
```

Comment:
__[querly]__
* ### Use Pathname method instead

#### Exception
* Want to write this code.


<!-- Comment by [SideCI](https://sideci.com) -->